pkg/controllers/cluster/manager/cluster_manager.go

```diff
@@ -273,15 +273,35 @@ func (m *ClusterManager) handleInstanceNotFound(ctx context.Context, primaryAddr
 
 // bootstrap bootstraps the cluster. Called on the first Pod in the StatefulSet.
 func (m *ClusterManager) bootstrap(ctx context.Context) (*innodb.ClusterStatus, error) {
-	if err := clearBinaryLogs(ctx, m.Instance); err != nil {
-		return nil, errors.Wrap(err, "failed to clear binary logs")
-	}
+	msh := m.mysqlshFactory(m.Instance.GetShellURI())
 
-	glog.V(4).Infof("Creating the cluster on the primary instance")
-	status, err := m.mysqlshFactory(m.Instance.GetShellURI()).CreateCluster(ctx, m.Instance.MultiMaster)
+	status, err := msh.GetClusterStatus(ctx)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to create new cluster")
+		mshErr, ok := errors.Cause(err).(*mysqlsh.Error)
+		if !ok {
+			return nil, errors.Wrap(err, "getting cluster status")
+		}
+
+		if strings.Contains(mshErr.Message, "Cannot perform operation while group replication is starting up") {
```

Comment:
Not really that happy with all this string comparison. Suggestions welcome.

Reply:
Are the return codes different?